src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -724,6 +724,12 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 		return result;
 	}
 
+	public getNonDefaultOptions(profile: interfaces.IConnectionProfile): string {
+		let convProfile = new ConnectionProfile(this._capabilitiesService, profile);
```

Comment:
Did you mean `connProfile` instead of` convProfile`?

Reply:
It means converted profile, this is deliberate